### ISSUE TITLE
Broken Links on website and documentation

### ISSUE THREAD

**Issue by gemiusz** [gatling/gatling#4528]
You need check whole website and documentation for broken links - there may of them.
I have created PR for 2 of them but there are more to fix - visible and not visible but in page code e.g:  ```<a href="https://1234567890.gatling.io/pricing" rel="noopener"></a>```

PR:
https://github.com/gatling/gatling/pull/4526
https://github.com/gatling/gatling/pull/4527

**Comment by notdryft** [comment]
Good catch, but I was a bit surprised we didn't caught that one since internal links and anchors are supposed to be checked when the documentation builds itself. For example, it should have printed:

```console
ERROR [en] REF_NOT_FOUND: Ref "#run-the-test-locally": ".../src/docs/content/tutorials/scripting-intro/index.md:18:34": anchor not found
```

The link you found didn't use the shortcode at all but was a simple markdown link:

```markdown
[run your simulation locally](#run-the-simulation-locally)
 ```

Which should have been:

```markdown
[run your simulation locally]({{< ref "#run-the-simulation-locally" >}})
 ```

It turns out there were a lot of these so I went ahead and fixed them all! Thanks for catching.

**Comment by guilgaly** [comment]
I'll close this issue now, since we have also fixed the `https://1234567890.gatling.io` links on the main website on Friday, and checked the other external links from both the docs and main websites yesterday, so hopefully links should be OK now (unless we missed something). Thanks again for the heads up and for the pull requests!
